src/UI/Message.jsx

```diff
@@ -32,24 +34,43 @@ class Message extends Component {
 		} else {
 			return null;
 		}
+		const buttonRenderError = "_Unable to show unsupported button_";
 
 		return (
 			<div className={classNames}>
 				{
 					this.props.showAgent
 					&& this.props.message.agent
 					&& this.props.message.agent.name.length > 0
-						&& <div className={styles.name}>
-							{this.props.message.agent.name}
-						   </div>
+					&& <div className={styles.name}>
+						{this.props.message.agent.name}
+					</div>
 				}
 				<div className={styles.message}>
+					<ReactMarkdown linkTarget={linkTarget} remarkPlugins={[gfm]} skipHtml={true}>
+						{this.props.message.message}
+					</ReactMarkdown>
+					{
+						this.props.message.buttons
+						&& this.props.message.buttons.map((button, index) => {
+							switch (button.type) {
+							case MessageButtonTypes.Reply:
+								// eslint-disable-next-line max-len,react/no-array-index-key
+								return <ReplyButton className={styles.button} key={index} onClick={this.props.onReplyButtonClick} payload={button.payload} title={button.title || button.payload} />;
+							case MessageButtonTypes.WebUrl:
+								// eslint-disable-next-line max-len,react/no-array-index-key
+								return <WebUrlButton className={styles.button} key={index} payload={button.payload} title={button.title || button.payload} />;
+							case MessageButtonTypes.PhoneNumber:
+								// eslint-disable-next-line max-len,react/no-array-index-key
+								return <CallButton className={styles.button} key={index} payload={button.payload} title={button.title || button.payload} />;
```

Comment:
Waarom de fallback voor de titel onderdeel van de Message logica laten zijn. Ik zou m juist in de button zelf verwachten. Dat payload required is en title optioneel. En als title dus niet gezet of leeg is je dan de payload toont.

Sowieso hoe gaat je render nu om met een lege string als titel. Pakt hij dat ook payload? Ik zou dat wel verwachten maar denk niet dat dit nu zo werkt. 

_Dit geld ook voor de andere 3 buttons, maar even op aparte regel gezet voor t overzicht_

Reply:
> Waarom de fallback voor de titel onderdeel van de Message logica laten zijn. Ik zou m juist in de button zelf verwachten. > Dat payload required is en title optioneel. En als title dus niet gezet of leeg is je dan de payload toont.

Done, 8fef62cce58fa5f777fd55c41d59d5b67c948253

> Sowieso hoe gaat je render nu om met een lege string als titel. Pakt hij dat ook payload? Ik zou dat wel verwachten maar denk niet dat dit nu zo werkt.

Daar is deze test voor: https://github.com/parley-messaging/web-library/blob/90bc52e18974f750f9d036a6ba784f634aceb264/cypress/e2e/ui_spec.js#L650-L692